tests/test_indexer.py

```diff
@@ -10,20 +10,24 @@
 from typing import Dict
 
 import boto3
-import moto
 from botocore.exceptions import ClientError
-from elasticsearch import Elasticsearch
+
 
 import dss
-from dss import BucketStage, Config, DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE
-from dss.events.handlers.index import process_new_indexable_object, ElasticsearchClient
+import moto
```

Comment:
We group imports into 3 categories:
- stdlib
- third party
- our own libraries
Each group is separated by a blank line.

Since moto is a third-party library, please move it back into the block above.